charter.html

```diff
@@ -252,14 +252,22 @@ <h3>Timeline</h3>
 
 	<section id="success-criteria">
 	  <h2>Success Criteria</h2>
-	  <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
-
-	  <p>
-          Each normative specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+	  <p>The WG will progress its normative specifications through the following
+    standardization process: 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">First Public
+      Working Draft</a>, 
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">Working
+      Draft</a>, and
+    <a href="https://www.w3.org/2021/Process-20211102/#RecsCR">Candidate
+      Recommendation</a>. 
+    It is expected that to reach the Candidate Recommendation stage, each normative
+    specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
```

Comment:
Here's the rough change I was suggesting in today's meeting, to ensure there's a clear phase (CR Draft) where the WG says "this is ready for implementation" that doesn't itself require the implementations to have already happened.

```suggestion
    <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">Working
      Draft</a>,
    <a href="https://www.w3.org/2021/Process-20211102/#candidate-recommendation-draft">Candidate
      Recommendation Draft</a>, and
    <a href="https://www.w3.org/2021/Process-20211102/#candidate-recommendation-snapshot">Candidate
      Recommendation Snapshot</a>.
    It is expected that to reach the Candidate Recommendation Snapshot stage, each normative
    specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
```

@wseltzer @martinthomson @cwilso, what do you think?

It's probably worth raising this overall question with the Process CG, but that needn't block progress on this charter.

Reply:
The one question I have is whether this needs to address the * on the [slide](https://www.w3.org/2020/Talks/TPAC/W3C2020/?full#7) to say there may be "0 or more".

Reply:
This seems fine to me.  I don't think that we need to worry about the 0 or more piece here.  I expect we'll use all of those states.  I can't see us creating a snapshot and then walking away with no further drafts.

Reply:
I am fine with that just wanted to get an explicit "nope not needed" from somebody.

Reply:
Works for me

Reply:
LGTM.